zebra-network/src/protocol/external/codec.rs

```diff
@@ -686,4 +694,71 @@ mod tests {
         assert_eq!(format!("{:?}", decode_state),
                    "DecodeState::Body { body_len: 43, command: \"v�ion\\u{0}\\u{0}\\u{0}\\u{0}\\u{0}\", checksum: Sha256dChecksum(\"bafaa2e3\") }");
     }
+
+    #[test]
+    fn max_msg_size_round_trip() {
+        use std::sync::Arc;
```

Comment:
```suggestion
        use std::sync::Arc;
        zebra_test::init();
```

Reply:
Added this too at https://github.com/ZcashFoundation/zebra/commit/d60ccac5f7902a4f0ce24c1f6de0b6f71c95a7ac